aries_vcx/src/handlers/connection/invitee/state_machine.rs

```diff
@@ -504,27 +472,6 @@ pub mod test {
             }
         }
 
-        mod get_thread_id {
-            use super::*;
-
-            #[test]
-            #[cfg(feature = "general_test")]
-            fn handle_response_fails_with_incorrect_thread_id() {
```

Comment:
I think this test should still be valid/working? If we try to process connection response with wrong thread id, we should detect it

Reply:
As I said in another comment, I disagree with verifying thread id multiple times or in every handler.